Cargo.lock

```diff
@@ -2657,7 +2658,6 @@ dependencies = [
 [[package]]
 name = "typst-dev-assets"
 version = "0.11.0"
-source = "git+https://github.com/typst/typst-dev-assets?rev=ee8ae61cca138dc92f9d818fc7f2fc046d0148c5#ee8ae61cca138dc92f9d818fc7f2fc046d0148c5"
```

Comment:
strange that it removes the information from the lock file

Reply:
Weird, it got readded now, not sure what happened.